src/assets/styles/tableMobile.css

```diff
@@ -32,7 +33,7 @@
 	#table-mobile td, 
 	#table-mobile tr { 
 		display: block;
-		padding: 10px 0;
+		/* padding: 10px 0; */
```

Comment:
Do we really need to remove that spacing entirely? 5px or less might look better

Reply:
Agree. Made it 5 px.